### DISCUSSION TITLE
Help Installing?

### DISCUSSION THREAD

**Discussion by Musicmonk84** [publicsamples/Oi-Grandad#8] [Q&A]
Hi, I'm new to GitHub and Oi-Grandad. I don't know how to install this thing. The most recent release doesn't have a DMG so I'm kind of lost. Thanks.

**Comment by OneShortMoment** [comment]
You probably missed the additional "Presets & Samples" download (like myself and some other users posting in the comments here...)
Have a look here, where I described how I got it to work (on Windows, but I assume it should be the same on a Mac if you exchange your user folder name for what's appropriate there): 
https://github.com/publicsamples/Oi-Grandad/discussions/13#discussioncomment-4210249

**Comment by RickTaylor79** [comment]
Hey, sorry for not responding! I didn't have my notifications set up properly. 

The latest release should fix all installation issues.
